components/codec/src/byte.rs

```diff
@@ -26,6 +29,29 @@ impl MemComparableByteCodec {
         (src_len / MEMCMP_GROUP_SIZE + 1) * (MEMCMP_GROUP_SIZE + 1)
     }
 
+    /// Gets the first encoded bytes' length in memory-comparable-encoded data.
+    ///
+    /// Memory-comparable-encoding includes a VarInt encoded length prefix (1 ~ 9 bytes) and N bytes payload.
+    /// This function gets the total bytes length of memory-comparable-encoded data, including the length prefix.
+    ///
+    /// Note:
+    ///     - This function won't check whether the bytes are encoded correctly.
+    ///     - There can be multiple memory-comparable-encoded data, placed one by one. This function only returns
+    ///       the length of the first one.
+    pub fn encoded_bytes_len(encoded: &[u8], desc: bool) -> usize {
+        let mut idx = MEMCMP_GROUP_SIZE;
+        loop {
+            if encoded.len() < idx + 1 {
+                return encoded.len();
+            }
+            let marker = encoded[idx];
+            if (desc && marker != 0) || (!desc && marker != !0u8) {
```

Comment:
I believe it should be:

```rust
if (desc && marker != ENC_MARKER) || (!desc && marker != !ENC_MARKER) {
```

Reply:
I this is copy from `util::codec::bytes`
https://github.com/tikv/tikv/blob/4b48f1acafe477c9dd52720a36cfea326593b577/src/util/codec/bytes.rs#L162
do you mean there is logic error or you suggest to make  `ENC_MARKER=0u8`(the old ENC_MARKER is b'\xff)?

Reply:
Sorry I made a mistake. Your code is fine!